openpype/hosts/houdini/plugins/publish/collect_review_data.py

```diff
@@ -17,6 +17,7 @@ def process(self, instance):
         # which isn't the actual frame range that this instance renders.
         instance.data["handleStart"] = 0
         instance.data["handleEnd"] = 0
+        instance.data["fps"] = hou.fps()
```

Comment:
Looks good to me! Note that there's also [CollectWorksceneFPS](https://github.com/ynput/OpenPype/blob/7b2932c9a78aa0e829a6bd06b148eb29a6f31e7d/openpype/hosts/houdini/plugins/publish/collect_workscene_fps.py#L5) collector and thus we could also do `instance.context.data["fps"]`.

I'm fine with either way however. Maybe @antirotor or @iLLiCiTiT has a preference.

Reply:
I agree, in case of review we can safely use the value from context. I hope we'll never find use cases where you would need to set different fps in one work file per instance :)